hooks/ad-join-relation-joined.ps1

```diff
@@ -22,27 +22,28 @@ try {
     Import-Module JujuHooks
     Import-Module JujuUtils
 
+    $renameReboot = Rename-JujuUnit
```

Comment:
I didn't check yet the `2.x` OpenStack hostname don't have long prefixes. But we should totally do that, if we are safe with hostnames already given by Juju.